diff --git a/node_modules/smee-client/index.js b/node_modules/smee-client/index.js
index 744cb0c..4e38100 100644
--- a/node_modules/smee-client/index.js
+++ b/node_modules/smee-client/index.js
@@ -1,3 +1,4 @@
+// See https://github.com/probot/smee-client/pull/152
 "use strict";
 const validator = require("validator");
 const EventSource = require("eventsource");
@@ -25,10 +26,12 @@ class Client {
         target.search = querystring.stringify(mergedQuery);
         delete data.query;
         const req = superagent.post(url.format(target)).send(data.body);
+        const bodyLength = JSON.stringify(data.body).length;
         delete data.body;
         Object.keys(data).forEach(key => {
             req.set(key, data[key]);
         });
+        req.set('content-length', `${bodyLength}`);
         req.end((err, res) => {
             if (err) {
                 this.logger.error(err);
